scripts/covid19_india/medical_tests_in_data/COVID19_tests_india.tmcf

```diff
@@ -8,4 +8,4 @@ value: C:COVID19_tests_india->CumulativeCount_MedicalTest_ConditionCOVID_19
 
 Node: E:COVID19_tests_india->E1
 typeOf: schema:Country
-isoCode: C:COVID19_tests_india->isoCode
\ No newline at end of file
+countryAlpha2Code: C:COVID19_tests_india->isoCode
```

Comment:
Might need to add it to http://google3/datacommons/import/mcf_vocab.h?l=234-236&rcl=333437561 too?